src/main/java/com/mojang/brigadier/arguments/StringArgumentType.java

```diff
@@ -147,9 +147,9 @@ private static String escape(final String input) {
      */
     public enum StringType {
         /**
-         * Matches a single word, optionally concatenated with underscores.
-         * Can contain the following characters: <br>
-         * {@code [a-zA-Z0-9_\-.+]}
+         * Matches a single word.
+         * Can contain the characters outline in {@link StringReader#isAllowedInUnquotedString(char)}.
```

Comment:
"outline" -> "outlined"?

Reply:
I just forgot to delete that one, sorry :/

And yes.